Unix/regress

```diff
@@ -228,7 +253,7 @@ cd omi-$version
 ## Create the prefix directory:
 ##
 
-prefix=/tmp/OMI932E75578CAB46E4A3EB87787B9EA40F
+prefix=/tmp/OMI932E75578CAB46E4A3EB87787B9EA40F-$USER
```

Comment:
Okay, I'll bite. What the heck is `932E75578CAB46E4A3EB87787B9EA40F`? Perhaps a comment is in order here ...